msr_rr_cfp.md

```diff
@@ -0,0 +1,89 @@
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+* **Example**:: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
+
+# Structured Abstract (required)
+The abstract should describe in 200 words or so:
+
+## Background/Context
+What is your research about? Why are you doing this research, why is it interesting?
+
+**Example**: The enterprise is the flag ship of the federation, and it allows families to travel onboard. However, there are no studies that evaluate how this affects the crew members”
+
+## Objective/Aim
+What exactly are you evaluating? What are the objects of the study?
+
+**Example**: We are studying whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family
+
+## Method
+How are you addressing your objective? What data sources are you using.
+**Example**: We conduct an observational study and use a between subject design. To analyze the data, we use a t test or Wilcoxon test, depending on the underlying distribution. Our data come computer monitoring of Enterprise crew members.
+
+## Early Results
+Here come the pilot data, if you have any. Give the reviewer some indication that you have reason to believe this research plan is promising.
```

Comment:
I'm a bit worried that "Early Results" suggests that this is just another NIER / ERA track. Is this really needed, and if so, would it make sense to make it very explicit that this is not needed for acceptance?

Reply:
maybe "preliminary results"? the purpose is to show a proof of concept/plan. 

(Incidentally my feeling is that this is exactly what a NIER paper should be)

Reply:
Right. And to de-emphasise this aspect, I'd probably suggest to not even mention results in the CfP. Not sure about MSR, but at ICSME we'll have NIER and RR -- the lack of results and 100% focus on methodology for RR could be what sets the two apart.

Reply:
ok, we can probably handle cases where the reviewer is unconvinced by the potential of the study in the rebuttal phase, where preliminary results could be used.

Reply:
And pick reviewers who don't expect results in an RR submission :)

Reply:
I also think that (even early) results should not be expected in an RR submission. I would expect an RR reviewer to mainly focus on assessing the soundness of the hypothesis/RQs and the soundness of the proposed methodology. While early results in NIER papers are useful to strengthen the authors' vision, i.e. by demonstrating feasibility of the new ideas, potential impact,  etc., I think one of the added values of the RR track is to have the possibility to get a feedback on the methodology and study design itself by mitigating the reviewers' bias that could emerge due to, for example, negative results.

Reply:
remove early results section, see fc825ffc2a3a0185564e900e0d726153f3d64f73